__tests__/OrderedSet.ts

```diff
@@ -59,4 +59,80 @@ describe('OrderedSet', () => {
       'CD',
     ]);
   });
+
+  it('ensure that `subtract` works correcly (see https://github.com/immutable-js-oss/immutable-js/issues/139 )', () => {
+    const fillArray = nb =>
+      Array(nb)
+        .fill(1)
+        .map((el, i) => i + 1);
+
+    const capacity = 32;
+    // items from keys 0 to 31 and values 1 to 32
+    const defaultItems = fillArray(capacity);
+
+    const allItems = OrderedSet(defaultItems);
+
+    const partialCapacity = Math.ceil(capacity / 2) + 1;
+    const someOfThem = fillArray(partialCapacity);
+    expect(someOfThem.length).toBe(17);
+
+    const existingItems = OrderedSet(someOfThem).intersect(allItems);
+
+    expect(allItems.subtract(existingItems).size).toBe(15);
+    expect(allItems.subtract(existingItems).size + someOfThem.length).toBe(32);
+  });
+
+  it('ensure that `subtract` works correctly (see https://github.com/immutable-js-oss/immutable-js/issues/96 )', () => {
```

Comment:
```suggestion
  it('handles consecutive subtract invocations (https://github.com/immutable-js-oss/immutable-js/issues/96 )', () => {
```

Reply:
now that i’ve reviewed more of this PR, i’m thinking maybe this test case should be moved to `__tests__/issues.ts`

Reply:
Personally I'm not a big fan of a file named `issues`. For me, things related to `OrderedSet` should be in the `OrderedSet` file.
I will change for your better comment though.